Sming/Core/Network/Http/HttpServerConnection.cpp

```diff
@@ -64,6 +65,10 @@ int HttpServerConnection::onMessageComplete(http_parser* parser)
 		bodyParser(request, nullptr, PARSE_DATAEND);
 	}
 
+	if(bodyContentError) {
+		response.code = HTTP_STATUS_BAD_REQUEST; // preset error response
```

Comment:
Highly recommended: remove the comment as it does not add additional information about the logic and the flow.

Reply:
done